pkg/stack/unwind/unwind_table.go

```diff
@@ -44,11 +49,102 @@ func (t UnwindTable) Len() int           { return len(t) }
 func (t UnwindTable) Less(i, j int) bool { return t[i].Loc < t[j].Loc }
 func (t UnwindTable) Swap(i, j int)      { t[i], t[j] = t[j], t[i] }
 
+// processMaps returns a map of file-backed memory mappings for a given
+// process which contains at least one executable section. The value of
+// mapping contains the metadata for the first mapping for each file, no
+// matter if it's executable or not.
+//
+// This is needed as typically the first mapped section for a dynamic library
+// is not executable, as it may contain only data, such as the `.bss` or the
+// `.rodata` section.
+func processMaps(pid int) (map[string]*procfs.ProcMap, string, error) {
```

Comment:
I like this. AFAIR, we weren't doing this previously in other branch. :)

Reply:
Don't remember where I added the change, but yeah this was necessary as otherwise every single unwind row for dynamic libraries was wrong. Opened https://github.com/parca-dev/parca-agent/issues/845 to track this